src/renderer.ts

```diff
@@ -156,6 +158,632 @@ export class DefaultCellRenderer implements ICellRendererFactory {
 /**
  * a renderer rendering a bar for numerical columns
  */
+
+class HeatmapCellRenderer extends DefaultCellRenderer {
+
+  createSVG(col: Column, context: IDOMRenderContext): ISVGCellRenderer {
```

Comment:
request the Column type to have proper typing and access to custom properties and functions